tests/test.rs

```diff
@@ -2201,6 +2201,9 @@ fn test_oom_allocation() {
     let ws = xls.worksheets();
     assert_eq!(ws.len(), 1);
     assert_eq!(ws[0].0, "Colsale (Aug".to_string());
+
+    let path = test_path("OOM_alloc3.xls");
```

Comment:
Same comment as the higher level conversation: failing test cases should be merged with the commit that fixes them.